hierarchicalforecast/utils.py

```diff
@@ -476,12 +475,12 @@ def quantiles_to_outputs(quantiles:Iterable[float]):
 # %% ../nbs/utils.ipynb 37
 # given input array of sample forecasts and inptut quantiles/levels, 
 # output a Pandas Dataframe with columns of quantile predictions
-def samples_to_quantiles_df(samples:np.ndarray, 
-                            unique_ids:Iterable[str], 
-                            dates:Iterable, 
-                            quantiles:Optional[Iterable[float]] = None,
-                            level:Optional[Iterable[int]] = None, 
-                            model_name:Optional[str] = "model"):
+def samples_to_quantiles_df(samples: np.ndarray, 
+                            unique_ids: List[str], 
```

Comment:
I think these are better as `collections.abc.Sequence`

Reply:
Thanks - I looked it up and it's because it's a slightly more generic container?